uber/configspec.ini

```diff
@@ -383,6 +377,9 @@ prereg_hotel_info_link = string(default='http://magfest.org/bookhotel/')
 #
 # Settings in this section are automatically converted to global variable datetime objects,
 # locatized to the timezone specified in the above EVENT_TIMEZONE setting.
+#
+# Dates are in YYYY-MM-DD format. Time may also be specified by using YYYY-MM-DD HH. If a
```

Comment:
I was looking for whether we filed an issue about the fact that we wanted two badge printed deadlines but didn't have them, and found an issue about how no one can remember what the date settings mean (https://github.com/magfest/ubersystem/issues/755). I am also frequently unsure of what the date settings default to when you don't specify a time, so I finally added it to the documentation.